chia/timelord/timelord_state.py

```diff
@@ -110,7 +110,7 @@ def set_state(self, state: Union[timelord_protocol.NewPeakTimelord, EndOfSubSlot
         reward_challenge: Optional[bytes32] = self.get_challenge(Chain.REWARD_CHAIN)
         assert reward_challenge is not None  # Reward chain always has VDFs
         self.reward_challenge_cache.append((reward_challenge, self.total_iters))
-        log.info(f"Updated timelord peak to {reward_challenge}, total iters: {self.total_iters}")
+        log.info(f"Updated timelord peak to {bytes.hex(reward_challenge)}, total iters: {self.total_iters}")
```

Comment:
`reward_challenge.hex()` I think should work here

Reply:
Tried `reward_challenge.hex()` on an Intel Mac and it fails to convert to hex without a discernible error.

`Updated timelord peak to b'H:\xcd\xf3\xed!\x89\xfb\xa0\x1e=\xfb\x0b\xf0\x964\x83\xa8!\xb0\xc4\xef]v\x8a\xaf\xcfm\xed\xf5ky', total iters: 43245004112118`

Reply:
Wait... may have a binary conflict on my machine...

Reply:
Yup. Binary node wasn't picking up my timelord changes. Manually running confirmed that `.hex()` is totally fine.